pasts.lv,"HTTPSConnectionPool(host='pasts.lv', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))","HTTPSConnectionPool(host='pasts.lv', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))"
patefon.co,"HTTPSConnectionPool(host='patefon.co', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='patefon.co', port=443): Read timed out. (read timeout=2)"
patmar.ee,"HTTPSConnectionPool(host='patmar.ee', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='patmar.ee', port=443): Read timed out. (read timeout=2)"
patro.cz,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
pcfarm.ro,"HTTPSConnectionPool(host='pcfarm.ro', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: UNSUPPORTED_PROTOCOL] unsupported protocol (_ssl.c:1131)')))","HTTPSConnectionPool(host='pcfarm.ro', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: UNSUPPORTED_PROTOCOL] unsupported protocol (_ssl.c:1131)')))"
pclab.pl,"HTTPSConnectionPool(host='pclab.pl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))","HTTPSConnectionPool(host='pclab.pl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))"
pctown.co.nz,"HTTPSConnectionPool(host='pctown.co.nz', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='pctown.co.nz', port=80): Read timed out. (read timeout=2)"
pdamobiz.com,"HTTPSConnectionPool(host='news.pdamobiz.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='news.pdamobiz.com', port=443): Read timed out. (read timeout=2)"
pdn.ac.lk,"HTTPSConnectionPool(host='pdn.ac.lk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fced761ac10>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='pdn.ac.lk', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fced78a1b80>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
peatus.ee,"HTTPSConnectionPool(host='web.peatus.ee', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='web.peatus.ee', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
pedagogica.edu.co,"HTTPSConnectionPool(host='pedagogica.edu.co', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fced7a67700>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='pedagogica.edu.co', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fced75ef310>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
pediatriepropraxi.cz,"HTTPSConnectionPool(host='pediatriepropraxi.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))","HTTPSConnectionPool(host='pediatriepropraxi.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))"
pegem.net,"HTTPSConnectionPool(host='www.pegem.net', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.pegem.net', port=443): Read timed out. (read timeout=2)"
peika.bg,"HTTPSConnectionPool(host='www.peika.bg', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.peika.bg', port=443): Read timed out. (read timeout=2)"
penang.gov.my,"HTTPSConnectionPool(host='penang.gov.my', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fced96fedf0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='penang.gov.my', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fced9210fd0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
penzin.rs,"HTTPSConnectionPool(host='www.penzin.rs', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.penzin.rs', port=443): Read timed out. (read timeout=2)"
peoples.ru,"HTTPSConnectionPool(host='peoples.ru', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.peoples.ru', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: DH_KEY_TOO_SMALL] dh key too small (_ssl.c:1131)')))"
peoplesbank.lk,"HTTPSConnectionPool(host='peoplesbank.lk', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fced88205b0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='peoplesbank.lk', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fced7731ee0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
pepperfry.com,"HTTPSConnectionPool(host='www.pepperfry.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.pepperfry.com', port=443): Read timed out. (read timeout=2)"
perach.org.il,"HTTPSConnectionPool(host='perach.org.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='perach.org.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
perak.gov.my,"HTTPSConnectionPool(host='perak.gov.my', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fced68fef40>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='perak.gov.my', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fced6ff75b0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
persee.fr,"HTTPSConnectionPool(host='www.persee.fr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.persee.fr', port=443): Read timed out. (read timeout=2)"
pertanian.go.id,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
petah-tikva.muni.il,"HTTPSConnectionPool(host='petah-tikva.muni.il', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fced8d74430>, 'Connection to petah-tikva.muni.il timed out. (connect timeout=2)'))","HTTPConnectionPool(host='petah-tikva.muni.il', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fced62b7f40>, 'Connection to petah-tikva.muni.il timed out. (connect timeout=2)'))"
petitlyrics.com,"HTTPSConnectionPool(host='petitlyrics.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))","HTTPSConnectionPool(host='petitlyrics.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))"
petrzalka.sk,"HTTPSConnectionPool(host='petrzalka.sk', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.petrzalka.sk', port=443): Read timed out. (read timeout=2)"
pgr.gob.ni,"HTTPSConnectionPool(host='pgr.gob.ni', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fced6324bb0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='pgr.gob.ni', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fced993fd60>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
pharm16.gr,"HTTPSConnectionPool(host='www.pharm16.gr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.pharm16.gr', port=443): Read timed out. (read timeout=2)"
pharmilion.gr,"HTTPSConnectionPool(host='pharmilion.gr', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='pharmilion.gr', port=443): Read timed out. (read timeout=2)"
pharmindex-online.hu,"HTTPSConnectionPool(host='pharmindex-online.hu', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fceda63c2b0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='pharmindex-online.hu', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fced8ec9bb0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
phimmoizz.net,"HTTPSConnectionPool(host='phimmoizz.net', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='phimmoizz.net', port=80): Read timed out. (read timeout=2)"
phoneindex.co.il,"HTTPSConnectionPool(host='phoneindex.co.il', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='phoneindex.co.il', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fced8a603d0>, 'Connection to phoneindex.co.il timed out. (connect timeout=2)'))"
pichinchacomunicaciones.com.ec,"HTTPConnectionPool(host='www.pichinchacomunicaciones.com.ec', port=80): Read timed out.","HTTPConnectionPool(host='www.pichinchacomunicaciones.com.ec', port=80): Read timed out."
pikabu.ru,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
pikiwiki.org.il,"HTTPSConnectionPool(host='www.pikiwiki.org.il', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.pikiwiki.org.il', port=443): Read timed out. (read timeout=2)"
pikk.ee,"HTTPSConnectionPool(host='www.pikk.ee', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.pikk.ee', port=443): Read timed out. (read timeout=2)"
piluli.ru,"HTTPSConnectionPool(host='piluli.ru', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='piluli.ru', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
pirkka.fi,"HTTPSConnectionPool(host='pirkka.fi', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.pirkka.fi', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
pitat.com,"HTTPSConnectionPool(host='pitat.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fced85d51f0>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='pitat.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fced539ee20>: Failed to establish a new connection: [Errno 111] Connection refused'))"
pivnidenicek.cz,"HTTPSConnectionPool(host='pivnidenicek.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'pivnidenicek.cz' doesn't match 'www.pivnidenicek.cz'"")))","HTTPSConnectionPool(host='pivnidenicek.cz', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'pivnidenicek.cz' doesn't match 'www.pivnidenicek.cz'"")))"
pizzasnella.it,"HTTPSConnectionPool(host='pizzasnella.it', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fced85d5f70>: Failed to establish a new connection: [Errno 111] Connection refused'))","HTTPConnectionPool(host='www.pizzasnella.it', port=80): Read timed out. (read timeout=2)"
planalto.gov.br,"HTTPSConnectionPool(host='planalto.gov.br', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1131)')))","(""Connection broken: ConnectionResetError(104, 'Connection reset by peer')"", ConnectionResetError(104, 'Connection reset by peer'))"
pleasantgroveumc.net,"HTTPSConnectionPool(host='pleasantgroveumc.net', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fced1d42bb0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='pleasantgroveumc.net', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fced858bbb0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
plovdiv.bg,"HTTPSConnectionPool(host='plovdiv.bg', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))","HTTPSConnectionPool(host='plovdiv.bg', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))"
pmda.go.jp,"HTTPSConnectionPool(host='pmda.go.jp', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fced8952640>, 'Connection to pmda.go.jp timed out. (connect timeout=2)'))","HTTPConnectionPool(host='pmda.go.jp', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fceda2942b0>, 'Connection to pmda.go.jp timed out. (connect timeout=2)'))"
pnc.gob.sv,"HTTPSConnectionPool(host='pnc.gob.sv', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fced8caefd0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='pnc.gob.sv', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fced1ba8670>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
pnky.sk,"HTTPSConnectionPool(host='pnky.sk', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.pnky.sk', port=443): Read timed out. (read timeout=2)"
pnp.co.za,"HTTPSConnectionPool(host='pnp.co.za', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fced3fedc40>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='pnp.co.za', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fced65f4d30>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
poder-judicial.go.cr,"HTTPSConnectionPool(host='poder-judicial.go.cr', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fced782cbe0>, 'Connection to poder-judicial.go.cr timed out. (connect timeout=2)'))","HTTPConnectionPool(host='poder-judicial.go.cr', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fced5426430>, 'Connection to poder-judicial.go.cr timed out. (connect timeout=2)'))"
poderjudicial.gob.ni,"HTTPSConnectionPool(host='poderjudicial.gob.ni', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fced91ccdc0>, 'Connection to poderjudicial.gob.ni timed out. (connect timeout=2)'))","HTTPConnectionPool(host='poderjudicial.gob.ni', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fced91cce20>, 'Connection to poderjudicial.gob.ni timed out. (connect timeout=2)'))"
poduckun.net,"HTTPSConnectionPool(host='poduckun.net', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='poduckun.net', port=443): Read timed out. (read timeout=2)"
poipeplekkers.be,"HTTPSConnectionPool(host='poipeplekkers.be', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fced212edc0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='poipeplekkers.be', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fced858b940>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
poisci.me,"HTTPSConnectionPool(host='poisci.me', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='poisci.me', port=80): Read timed out. (read timeout=2)"
polazak.rs,Exceeded 30 redirects.,Exceeded 30 redirects.
police.gov.sg,"HTTPSConnectionPool(host='www.police.gov.sg', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.police.gov.sg', port=443): Read timed out. (read timeout=2)"
policia.gob.ni,"HTTPSConnectionPool(host='policia.gob.ni', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='policia.gob.ni', port=443): Read timed out. (read timeout=2)"
policia.gov.co,"HTTPSConnectionPool(host='policia.gov.co', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fced00b0580>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='policia.gov.co', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fced4249eb0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
politikaplus.com,"HTTPSConnectionPool(host='politikaplus.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='politikaplus.com', port=80): Read timed out. (read timeout=2)"
polito.it,"HTTPSConnectionPool(host='polito.it', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fced2b63c10>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='polito.it', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fced9adf850>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
polska.travel,"HTTPSConnectionPool(host='www.polska.travel', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.polska.travel', port=443): Read timed out. (read timeout=2)"
polsl.pl,"HTTPSConnectionPool(host='polsl.pl', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fced5d3a6a0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))","HTTPConnectionPool(host='polsl.pl', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fceda8514f0>: Failed to establish a new connection: [Errno 101] Network is unreachable'))"
pom.go.id,"HTTPSConnectionPool(host='pom.go.id', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='pom.go.id', port=443): Max retries exceeded with url: /new/ (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
pontofrio.com.br,"HTTPSConnectionPool(host='www.pontofrio.com.br', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.pontofrio.com.br', port=443): Read timed out. (read timeout=2)"
popcornfor2.com,"HTTPSConnectionPool(host='popcornfor2.com', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='popcornfor2.com', port=80): Read timed out. (read timeout=2)"
popularonline.com.my,"HTTPSConnectionPool(host='popularonline.com.my', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='popularonline.com.my', port=443): Read timed out. (read timeout=2)"
poradnikogrodniczy.pl,"HTTPSConnectionPool(host='poradnikogrodniczy.pl', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fced8c4ad00>, 'Connection to poradnikogrodniczy.pl timed out. (connect timeout=2)'))","HTTPConnectionPool(host='poradnikogrodniczy.pl', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fced9821640>, 'Connection to poradnikogrodniczy.pl timed out. (connect timeout=2)'))"
porezna-uprava.hr,"HTTPSConnectionPool(host='porezna-uprava.hr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fced9fa3460>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='porezna-uprava.hr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fced01bbeb0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
port2port.co.il,"HTTPSConnectionPool(host='port2port.co.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))","HTTPSConnectionPool(host='www.port2port.co.il', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: WRONG_SIGNATURE_TYPE] wrong signature type (_ssl.c:1131)')))"
portal.gov.bd,"HTTPSConnectionPool(host='portal.gov.bd', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))","HTTPSConnectionPool(host='portal.gov.bd', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))"
portaldasfinancas.gov.pt,"HTTPSConnectionPool(host='portaldasfinancas.gov.pt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fced8272a00>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='portaldasfinancas.gov.pt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fced66005b0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
portorecanaticalcio.it,"HTTPSConnectionPool(host='www.portorecanaticalcio.it', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.portorecanaticalcio.it', port=443): Read timed out. (read timeout=2)"
posted.co.rs,"HTTPSConnectionPool(host='posted.co.rs', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fced48a2ee0>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))","HTTPConnectionPool(host='posted.co.rs', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fced75efd60>: Failed to establish a new connection: [Errno -5] No address associated with hostname'))"
pozitivke.net,"HTTPSConnectionPool(host='pozitivke.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'pozitivke.net' doesn't match either of '*.mojhost.net', 'mojhost.net'"")))","HTTPConnectionPool(host='pozitivke.net', port=80): Read timed out. (read timeout=2)"
